lib/statsd.rb

```diff
@@ -328,17 +319,16 @@ def format_event(title, text, opts={})
 
   private
 
-  def escape_event_content(msg)
```

Comment:
Can we keep imperative verbs for method names, i.e `do_something` rather than `something_done` ?